File: Practical 2  - searching/python code/linear_search.py
arr = [10,20,30,60,55,35,76]
num = int(input("Enter the number to search: "))

found = False

for i in range(len(arr)):
    if arr[i]==num:
        found= True
        print(f"Number {num} found at index {i}")

if not found:
    print(f"Number {num} not found")

File: Practical-02/C code/linear_search.c
#include <stdio.h>

int main()
{
    int arr[100], n, num;
    int found = 0;

    printf("Enter the number of elements: ");
    scanf("%d", &n);

    printf("Enter the array elements:\n");
    for(int i = 0; i < n; i++)
    {
        scanf("%d", &arr[i]);
    }

    printf("Enter the element to search: ");
    scanf("%d", &num);

    // Linear Search
    for(int i = 0; i < n; i++)
    {
        if(arr[i] == num)
        {
            printf("Element found at index %d", i);
            found = 1;
            break;
        }
    }

    if(found == 0)
    {
        printf("Element not found");
    }

    return 0;
}